### DISCUSSION TITLE
発動の追加 (議論の必要がありそうなもの)

### DISCUSSION THREAD

**Discussion by Hourier** [hengband/hengband#1696] [Ideas]
★の追加依頼はあったものの、現時点で発動IDがなく却下したもの
個人的には全て恒久的却下でも良いような気もするが、意見を募集するためDiscussionとする
(特に超隠密はプレイヤーの状態管理コストが実装リターンに見合わない印象)
- 超隠密
- カマイタチ (自然領域の魔法)
- 魔神斬り (武芸)
- 矢弾作成

**Comment by RRyukkuriRyu** [comment]
超隠密と矢弾作成は職業のアイデンティティを潰しかねないので恒久的却下でいいと思います。
魔神斬りは剣術家でも殆ど使われていないので実装しても問題は無さそうに感じますが、原案の方は発動でドラゴン火炎にしてもしっくりくるかと。
カマイタチは使ったことが無いので分かりませんが、説明文を読んだ限りだとただの射程1の全方位攻撃に見えるので実装しても問題なさそうな気がします。

**Comment by Hourier** [comment]
ご意見ありがとうございます
他の方からの意見も引き続き待ちつつ、以下コメントします
- カマイタチは採用する方針で考えてみます
- 魔神斬り → ドラゴン火炎は前向きに検討します
- 残りの2つはやはり却下の方が良さそうです

**Comment by daradarach** [comment]
矢弾作成は元ネタ再現ですね。
ゲームへのバランス影響は、レアリティが十分に高ければ、Jを倒すために追尾の矢だけ集め続けることの代替程度かなと思ってます。
実装コストについては #1701 でやってみましたが問題あれば捨てて下さい。

**Comment by Hourier** [comment]
カマイタチは見落としていたようです
今もありました (但し特定★だけ別名にするのは変わらず却下)
矢弾作成はまだ議論中のようですので引き続きよろしくお願いします

**Comment by RRyukkuriRyu** [comment]
discordでの議論の結果、以下の点により矢弾生成は問題なさそうという感じになりました。
・あらゆる職業で追尾の矢が集められるといっても、射撃能力トップがアーチャーであることに変わりはない点
・レンジャーや盗賊、騎兵がミスリルの弾や追尾の矢を集めやすくなり、最終戦の準備で間延びすることが無くなるという点
・ダイアナの長弓のレアリティが高く、毎回安定して入手できるものではない点
これによりゲームバランスにはあまり影響を与えず、運よく入手できれば進行を快適にできる程度に収まるかと思われます。

**Reply by Hourier** [reply]
議論の取りまとめありがとうございます
矢弾生成に関しては実装案も着ているので、採用する方向で調整したいと思います

**Comment by meetname** [comment]
魔神斬りは発動効果にするよりは常時魔神斬り状態のアーティファクトを作ったほうが個性が出ると思われます。
重い武器に発動がつかないと性能が噛み合わず使われないでしょうし、剣術家でも使わないようなものを発動効果にしても最悪存在を忘れられるのでは。剣術家には無双三段という上位互換があるという理由もありますが。

**Reply by Hourier** [reply]
ソースコード全域に渡って特定1アーティファクトのための特殊処理を追加するだけの価値がある逸品には思えないので個性を出すのは却下します
